src/libcrun/handlers/krun.c

```diff
@@ -171,22 +171,6 @@ libkrun_configure_container (void *cookie, enum handler_configure_phase phase,
   int ret, rootfsfd;
   size_t i;
   struct krun_config *kconf = (struct krun_config *) cookie;
-  struct device_s kvm_device = { "/dev/kvm", "c", 10, 232, 0666, 0, 0 };
-  struct device_s sev_device = { "/dev/sev", "c", 10, 124, 0666, 0, 0 };
-  cleanup_close int devfd = -1;
-  cleanup_close int rootfsfd_cleanup = -1;
-  runtime_spec_schema_config_schema *def = container->container_def;
-  bool is_user_ns;
-  bool create_sev;
-
-  if (rootfs == NULL)
-    rootfsfd = AT_FDCWD;
-  else
-    {
-      rootfsfd = rootfsfd_cleanup = open (rootfs, O_PATH);
-      if (UNLIKELY (rootfsfd < 0))
-        return crun_make_error (err, errno, "open `%s`", rootfs);
-    }
```

Comment:
We need to keep this, as otherwise the `write_file_at()` will fail.

Reply:
I've dropped the patch for now, we can do this later